packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -115,28 +132,48 @@ interface OrganizationMongoSelectProps extends BaseMongoSelectProps {
   isOnPrem?: boolean;
 }
 
-const onKeyDown: React.KeyboardEventHandler = e => {
-  // Stops default browser behavior from automatically scrolling the component
-  if ([keyMap.ArrowUp, keyMap.ArrowDown].includes(e.keyCode)) {
-    e.preventDefault();
-  }
-};
-
 function OrgSelect({
   current,
   data,
   urls,
   isActive,
-  onChange,
+  onChange: onChangeProp,
   onClick,
   constructOrganizationURL,
   isOnPrem,
   disabled,
   loading = false,
 }: OrganizationMongoSelectProps) {
+  const [value, setValue] = useState('');
   const [open, setOpen] = useState(false);
   const onElementClick = useOnElementClick();
 
+  let renderedData = data;
+
+  const filterData = () => {
+    const search = new RegExp(`${value}`, 'i');
```

Comment:
Known bug if you typeL "\" it breaks, should I not turn the value into a regex?

Reply:
Can you not just do `new RegExp(String(value), 'i')` ?

Do we have tests for these cases?

Reply:
Yes the simplification works but still fails on \